x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/service/GetServiceAccountTokensRequestTests.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.security.action.service;
+
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.test.AbstractWireSerializingTestCase;
+
+import java.io.IOException;
+
+import static org.hamcrest.Matchers.containsString;
+
+public class GetServiceAccountTokensRequestTests extends AbstractWireSerializingTestCase<GetServiceAccountTokensRequest> {
+
+    @Override
+    protected Writeable.Reader<GetServiceAccountTokensRequest> instanceReader() {
+        return GetServiceAccountTokensRequest::new;
+    }
+
+    @Override
+    protected GetServiceAccountTokensRequest createTestInstance() {
+        final String namespace = randomAlphaOfLengthBetween(3, 8);
+        final String serviceName = randomAlphaOfLengthBetween(3, 8);
+        return new GetServiceAccountTokensRequest(namespace, serviceName);
+    }
+
+    @Override
+    protected GetServiceAccountTokensRequest mutateInstance(GetServiceAccountTokensRequest instance) throws IOException {
+        switch (randomIntBetween(0, 2)) {
+            case 0:
+                return new GetServiceAccountTokensRequest(
+                    randomValueOtherThan(instance.getNamespace(), () -> randomAlphaOfLengthBetween(3, 8)), instance.getServiceName());
```

Comment:
As above, you need to ensure this isn't the same as the one passed in (and other cases below)
```suggestion
                    randomValueOtherThan(instance.getNamespace(), () -> randomAlphaOfLength(10)), instance.getServiceName());
```

Reply:
This is wrapped inside a `randomValueOtherThan`. So I think we are good here.